geospaas_processing/copiers.py

```diff
@@ -88,5 +102,5 @@ def delete(self, ttl):
                 if ((entry.is_file(follow_symlinks=False) or entry.is_symlink())
                     and '.snapshot' not in entry.path
                     and entry.stat(follow_symlinks=False).st_uid == os.getuid()
-                    and time.time() - entry.stat(follow_symlinks=False).st_mtime > ttl*24*3600):
+                        and time.time() - entry.stat(follow_symlinks=False).st_mtime > ttl*24*3600):
```

Comment:
If you do that, please also add an indentation level in front of the other conditions (beginning with `and`).

Reply:
NO,I have not done that. It is from autopep8

Reply:
Done